FreeRTOS-Plus/Source/Utilities/mbedtls_freertos/mbedtls_bio_freertos_cellular.c

```diff
@@ -31,13 +31,16 @@
 
 /* FreeRTOS includes. */
 #include "FreeRTOS.h"
-#include "FreeRTOS_Sockets.h"
 
 /* Sockets wrapper includes. */
 #include "sockets_wrapper.h"
 
 /* mbed TLS includes. */
-#include "mbedtls_config.h"
```

Comment:
This uses a static config file name for MbedTLS. The changes for this are the recommended way for including MbedTLS config directly.